app/components/mixin-detail.js

```diff
@@ -17,6 +17,9 @@ export default Component.extend({
     return this.get('model.expand') && this.get('model.properties.length') > 0;
   }),
 
+  sortProperties: ['name'],
+  sortedProperties: sort('model.properties', 'sortProperties'),
```

Comment:
Can you please document these? We're trying to document new code to slowly transition to a documented codebase.

```js
/**
 * Used by the `sort` computed macro.
 *
 * @property sortProperties
 * @type {Array<String>}
 */
sortProperties: ['name'],

/**
 * Sort the properties by name to make them easier to find in the object inspector.
 *
 * @property sortedProperties
 * @type {Array<Object>}
 */
sortedProperties: sort('model.properties', 'sortProperties'),
```